script/testing/junit/sql/analyze.sql

```diff
@@ -0,0 +1,47 @@
+CREATE TABLE foo (a int, b int);
+
+SELECT a FROM foo WHERE a < 666;
+SELECT a FROM foo WHERE a <= 42;
+SELECT a FROM foo WHERE a > 999;
+SELECT a FROM foo WHERE a >= 24;
+SELECT a FROM foo WHERE a = 42;
+SELECT a FROM foo WHERE NOT a = 666;
+SELECT a FROM foo WHERE NOT a <> 666;
+SELECT a FROM foo WHERE b IS NULL;
+SELECT a FROM foo WHERE b IS NOT NULL;
```

Comment:
My goal here was to try and target different stats calculator selectivity operations with empty statistics and with full statistics and make sure it didn't blow up. That's why I have a bunch of different predicates. I included it here because it's somewhat related to ANALYZE. Do you think there's a better place I can move that type of test?

Reply:
Ohoh I see. That makes sense! Maybe add a bit comment to explain this?

Reply:
Done